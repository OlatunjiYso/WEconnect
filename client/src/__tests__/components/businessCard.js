import React from "react";

import BusinessCard from '../../components/businessCard';

describe('All tests for business body component', () => {
 
  const props = {
    business: { 
        image: '',
        name: '',
        city: '',
        state: '',
        id: 0
    }
  };
  
    it('should be defined', () => {
      expect(BusinessCard).toBeDefined();
    });
    it('should render the card', () => {
      const { wrapper } = shallow(<BusinessCard {...props} />);
      expect(wrapper).toMatchSnapshot();
    });
});
