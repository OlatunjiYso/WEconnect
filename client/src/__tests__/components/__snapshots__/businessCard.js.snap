// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All tests for business body component should render the card 1`] = `undefined`;
